docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -74,7 +74,7 @@ The following table lists the available GX Cloud Expectations.
 
 - **Batch column** - Select a name column from a prefilled list of DATE and DATETIME columns containing the date and time data.
 
-7. Complete the mandatory and optional fields for expectation.
+7. Complete the mandatory and optional fields for the expectation. A recurring validation schedule will be applied automatically to your expectation, based on the settings of your Expectation Suite.
```

Comment:
As suggested later (I spotted the second instance, but overlooked this one initially):

```suggestion
7. Complete the mandatory and optional fields for the Expectation. A recurring validation schedule will be applied automatically to your Expectation, based on the settings of your Expectation Suite.
```